aicsimageio/readers/reader.py

```diff
@@ -148,6 +148,30 @@ def get_physical_pixel_size(self, scene: int = 0) -> Tuple[float]:
         """
         return (1.0, 1.0, 1.0)
 
+    def get_channel_names(self, scene: int = 0) -> Optional[List[str]]:
+        """
+        Attempts to use the image's metadata to get the image's channel names.
+
+        Parameters
+        ----------
+        scene: int
+            The index of the scene for which to return channel names.
+
+        Returns
+        -------
+        channels_names: Optional[List[str]]
+            List of strings representing the channel names.
+            If channel dimension not present in file, return None.
+        """
+        # Check for channels dimension
+        if Dimensions.Channel not in self.dims:
+            return None
+
+        # Channel dimension in reader data, get default channel names
+        channel_index = self.dims.index(Dimensions.Channel)
+        channel_dim_size = self.dask_data.shape[channel_index]
```

Comment:
I forget, do readers have a .shape property?    Does using dask_data here have any weird implications on resolving the returned list?

Reply:
They do not have a shape property and the `dask_data` here simply constructs the delayed dask array but doesn't do any reading. (Besides the first plane read)